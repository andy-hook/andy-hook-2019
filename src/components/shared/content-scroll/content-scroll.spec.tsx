import React from "react"
import renderer from "react-test-renderer"
import { transitionState } from "@mock-data"

import ContentScroll from "./content-scroll"

describe("<Logo />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ContentScroll transitionState={transitionState} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
