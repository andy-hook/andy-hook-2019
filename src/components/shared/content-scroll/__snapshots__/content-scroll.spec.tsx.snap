// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders correctly 1`] = `
<div
  className="sc-bdVaJa fuwhFX"
>
  <div
    className="sc-bwzfXH dtqYqz"
  />
</div>
`;
